qa/rpc-tests/txn_doublespend.py

```diff
@@ -65,7 +65,7 @@ def run_test(self):
         if self.options.mine_block: expected += mining_reward
         expected += tx1["amount"] + tx1["fee"]
         expected += tx2["amount"] + tx2["fee"]
-        assert_equal(self.nodes[0].getbalance(), expected)
+        assert_equal(self.nodes[0].getbalance("*", 0), expected)
```

Comment:
This change should not be necessary if we're keeping 0 as the default for `minconf`.